.claude/commands/gh-bulk-issues.md

```
@@ -0,0 +1,88 @@
+1. **Create a git worktree** with a dedicated branch:
+   ```
+   git worktree add ../$(basename $PWD)-issue-<NUMBER> -b fix/issue-<NUMBER>
+   ```
+
+2. **Install and build** in the worktree. Run 2 worktrees at a time to manage CPU:
+   ```
+   cd ../$(basename $PWD)-issue-<NUMBER> && pnpm i && pnpm build
+   ```
+
+3. **Spawn an `mc` headless instance** in each worktree with a generous timeout (30 minutes):
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)